purchase_vendor_bill_breakdown/readme/USAGE.rst

```diff
@@ -0,0 +1,9 @@
+* Enable **Use Product Components** option in the **Contact** section of the  **Sales & Purchases** tab.
+* Enable **Use Product Components** option in the **Products** section of the **Purchase** tab.
+* In the **Purchase** tab of the **Product**  there will be a button added for the seller.
+* Click the button and add components for the product.
```

Comment:
* In vendor, enable **Use Product Components** option in the **Sales & Purchases** tab.
* In product, enable **Use Product Components** option in the **Purchase** tab.
* If both product and vendor have the flag activated, in the **Purchase** tab of the **Product**, table **Vendor** there will be a button at the end of the supplierinfo line.
* Click the button and add components and quantities to display in vendor bill for the product > save
* Supplierinfo price is now readonly and calculated as sum of the first applicable supplierinfo price * their quantities.